src/main/java/org/folio/marc4ld/service/ld2marc/processing/combine/impl/AbstractDataFieldCombiner.java

```diff
@@ -58,6 +80,28 @@ private boolean notContainsInCombined(char fieldTag) {
       .isEmpty();
   }
 
+  private boolean hasPrioryFromExist(char fieldTag, ResourceEdge resourceEdge) {
+    var currentEdge = containedFields.getOrDefault(fieldTag, startedEdge);
+    return getComparator(fieldTag)
+      .compare(resourceEdge, currentEdge) > 0;
+  }
+
+  private void replaceField(char fieldTag, DataFieldContainer dataFieldContainer) {
```

Comment:
replaceSubfield (looks like same for the method below)

Reply:
rollback approach